From 96bfc5b992d011a7c2845e7eaf4413f50ee662b6 Mon Sep 17 00:00:00 2001
From: Sathisha S <sathisha.shivaramappa@arm.com>
Date: Tue, 23 Sep 2025 10:40:41 +0000
Subject: [PATCH] Disable BBTestGetImageInfoConformanceTestCheckpoint4 test

This test is disabled due to a clarification pending in
UEFI Specification.
The test causes synchronous exception in some environment
---
 .../BlackBoxTest/FirmwareManagementBBTestConformance.c          | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/FirmwareManagement/BlackBoxTest/FirmwareManagementBBTestConformance.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/FirmwareManagement/BlackBoxTest/FirmwareManagementBBTestConformance.c
index 22e8dcf2..1b942df4 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/FirmwareManagement/BlackBoxTest/FirmwareManagementBBTestConformance.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/FirmwareManagement/BlackBoxTest/FirmwareManagementBBTestConformance.c
@@ -571,7 +571,7 @@ BBTestGetImageInfoConformanceTest (
   BBTestGetImageInfoConformanceTestCheckpoint1 (StandardLib, FirmwareManagement);
   BBTestGetImageInfoConformanceTestCheckpoint2 (StandardLib, FirmwareManagement);
   BBTestGetImageInfoConformanceTestCheckpoint3 (StandardLib, FirmwareManagement);
-  BBTestGetImageInfoConformanceTestCheckpoint4 (StandardLib, FirmwareManagement);
+  //BBTestGetImageInfoConformanceTestCheckpoint4 (StandardLib, FirmwareManagement);
 
   return EFI_SUCCESS;
 }
-- 
2.43.0

